website/templates/index/quickstart.html

```diff
@@ -20,10 +20,10 @@ <h2 id="quick-start" class="display-2 mb-5 mb-md-8 mb-lg-10 text-center">Quick S
                     <a class="nav-link" id="arm-tab" data-toggle="tab" href="#arm" role="tab" aria-controls="arm" aria-selected="false" title="ARM packages">Linux (ARM)</a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" id="mac-x86-tab" data-toggle="tab" href="#mac-x86" role="tab" aria-controls="mac-x86" aria-selected="false" title="Mac x86 packages">Mac (x86)</a>
+                    <a class="nav-link" id="mac-x86-tab" data-toggle="tab" href="#mac-x86" role="tab" aria-controls="mac-x86" aria-selected="false" title="Mac x86 packages">Mac (Intel)</a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" id="mac-arm-tab" data-toggle="tab" href="#mac-arm" role="tab" aria-controls="mac-arm" aria-selected="false" title="Mac ARM packages">Mac (ARM)</a>
+                    <a class="nav-link" id="mac-arm-tab" data-toggle="tab" href="#mac-arm" role="tab" aria-controls="mac-arm" aria-selected="false" title="Mac ARM packages">Mac (Apple Silicon)</a>
```

Comment:
```suggestion
                    <a class="nav-link" id="mac-arm-tab" data-toggle="tab" href="#mac-arm" role="tab" aria-controls="mac-arm" aria-selected="false" title="Mac ARM packages">macOS (Apple silicon)</a>
```

Reply:
Why "silicon" in lower case?

Reply:
That's how it is called officially:  https://support.apple.com/en-us/HT211814